dtfs-central-api/src/constants/payload.js

```diff
@@ -0,0 +1,12 @@
+const BANK = {
+  id: String,
```

Comment:
When you create a bank, does it already have the `id`?

Reply:
Yep it always will have one.